docs/articles/documentation/guides/basic-guides/best-practices.md

```diff
@@ -0,0 +1,351 @@
+---
+layout: docs
+title: Best Practices
+permalink: /documentation/guides/basic-guides/best-practices.html
+---
+# Best Practices
+
+This article describes the recommended ways to test with TestCafe and covers the following topics:
```

Comment:
I'd like to avoid using the word "covers". Can we change it to the following?

```suggestion
This article describes the recommended ways to test with TestCafe. The main topics are as follows:
```

Reply:
Applied